beetsplug/alternatives.py

```diff
@@ -410,6 +457,80 @@ def finalize_converted_item(item: Item, dest: Path):
                 for item, dest in _get_queue_available(converting_done):
                     finalize_converted_item(item, dest)
 
+            if self._config.album_art_copy:
+                self.update_art()
+
+    def update_art(self, link: bool = False):
+        for album in self.lib.albums():
+            if not self._config.query.match(album) and not any(
+                self._config.query.match(item) for item in album.items()
+            ):
+                continue
+
+            dest_dir = self.album_destination(album)
+            if not dest_dir:
+                continue
+
+            artpath = album.artpath and Path(str(album.artpath, "utf8"))
+            if not artpath or not artpath.is_file():
+                continue
+
+            dest = album.art_destination(album.artpath, bytes(str(dest_dir), "utf8"))
+            dest = Path(str(dest, "utf8"))
+
+            if self._config.album_art_format:
+                new_format = self._config.album_art_format.lower()
+                # A nonexhaustive map of image "types" to extensions overrides
+                new_format = {"jpeg": "jpg"}.get(new_format, new_format)
+
+                fname = dest.parent / dest.stem
+                dest = fname.with_suffix("." + new_format)
+
+            if dest.is_file() and not (artpath.stat().st_mtime > dest.stat().st_mtime):
+                continue
+
+            dest = bytes(str(dest), "utf8")
+            artpath = bytes(str(artpath), "utf8")
+
+            if link:
+                self._log.debug(f"Linking art from {album.artpath} into {dest_dir}")
+                util.link(artpath, dest, replace=True)
+            else:
+                path = self.resize_art(artpath)
+                self._log.debug(f"Copying art from {path} into {dest_dir}")
+                util.copy(path, dest, replace=True)
+
+            print_(f"~{dest_dir}")
```

Comment:
Shouldn’t this by `f"~{dest}"`. If so make sure that `dest` remains a `Path` and is not changes to `bytes`. Other wise the output will look like
```plain
~b'/home/....'
```